server/channels/store/sqlstore/post_store.go

```diff
@@ -3239,7 +3239,7 @@ func (s *SqlPostStore) GetPostReminders(now int64) (_ []*model.PostReminder, err
 
 func (s *SqlPostStore) GetPostReminderMetadata(postID string) (*store.PostReminderMetadata, error) {
 	meta := &store.PostReminderMetadata{}
-	err := s.GetReplicaX().Get(meta, `SELECT c.id as ChannelId,
```

Comment:
I can probably revert this commit.

Reply:
done.

Reply:
@agnivade Are you sure that the query doesn't need to get updated? @TealWater is changes the field names of `PostReminderMetadata`.

Reply:
We need this change, otherwise CI fails on MySql: 

```
=== FAIL: channels/store/localcachelayer TestPostStore/LocalCache+MySQL/GetPostReminderMetadata (0.01s)
    post_store.go:5007: 
        	Error Trace:	/mattermost/server/channels/store/storetest/post_store.go:5007
        	            				/mattermost/server/channels/store/storetest/post_store.go:65
        	Error:      	Received unexpected error:
        	            	missing destination name ChannelId in *store.PostReminderMetadata
        	            	failed to get post reminder metadata: postId bwoibdsqwfbozdyx5i1bu7p7uo
```

Reply:
Oh I see. I didn't realize that the sqlx parsing was case-sensitive. Yes, we need this then. Apologies for the double work.